variables.tf

```diff
@@ -73,6 +73,8 @@ variable secret_environment {
   type        = map(string)
   default     = {}
   description = <<EOF
+  DEPRECATED: Use a combination of `plain_secrets`, and `vault_secrets`.
```

Comment:
I would maybe not call this deprecated, as others may prefer SSM over Vault. Not a huge deal, as I don't think this module is used significantly by anyone else 🤷

Reply:
Yup, you're right. I originally intended for this to be replaced by `plain_secrets`, but then I realised I was unnecessarily duplicating work!

Can you also tell me how to run a release for this module?